include/utility/vpd_specific_utility.hpp

```diff
@@ -466,5 +466,89 @@ inline bool findCcinInVpd(const nlohmann::json& i_JsonObject,
     logging::logMessage("VPD type not supported. Can't find CCIN");
     return false;
 }
+
+/**
+ * @brief API to reset data of a FRU on DBus.
```

Comment:
Nit: It only resets the data under PIM, Can you add that to the Doxygen?
Or you can rename the API as well
resetDataUnderPim?

Reply:
Done
Updated the API name